SwiftCurrent.podspec

```diff
@@ -26,6 +26,12 @@ Pod::Spec.new do |s|
     ss.dependency 'SwiftCurrent/Core'
   end
 
+  s.subspec 'BETA_SwiftUI' do |ss|
+    ss.ios.deployment_target = '11.0'
```

Comment:
Curious: Why this is 11.0 when everything we're doing for SwiftUI is targeting at least 14.0?

Reply:
Take a look at the commit/action history. I started with 14, then CocoaPods cried cause the subspec had a higher version than the primary spec. I suspected there was no way around it, so downgraded to 11 to make the primary spec